Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -713,23 +713,6 @@ class BinaryIntegerTests : XCTestCase {
     }
 }
 
-class HashableTests : XCTestCase {
-    func hashableTests() -> [(input: UInt128, result: Int)] {
-        var tests = [(input: UInt128(), result: 0)]
-        tests.append((input: UInt128(1), result: 1))
-        tests.append((input: UInt128(Int.max), result: Int.max))
-        tests.append((input: try! UInt128("85070591730234615862769194512323794261"),
-                      result: -1537228672809129302))
-        return tests
-    }
-
-    func testHashValueProperty() {
-        hashableTests().forEach { test in
-            XCTAssertEqual(test.input.hashValue, test.result)
-        }
-    }
-}
-
```

Comment:
I would still like to have tests verifying that hashing works. Could you adapt these tests to work with the newer hashing API?

Reply:
The new `Hasher` API seems not giving us the opportunity to implement a deterministic hash value. I have no idea how to verify the correctness of the hash, do you have any ideas?

Reply:
That is quite annoying. I'm thinking a good approach would be to try change the test to compute the hash of let's say 10 disparate numbers, say some with just lower bits set, others with just higher, some with both with alternating bit patterns, and then compare that they all generate a different value. Then, perform a hash the same list 2+ times and make sure it generates the same value each time. How does that approach sound?